src/Discovery/DiscoveryMDSSearch.tsx

```diff
@@ -4,16 +4,19 @@ import {
 } from 'antd';
 import { SearchOutlined } from '@ant-design/icons';
 
-const DiscoveryMDSSearch = (props: { searchTerm, handleSearchChange}) => (
-  <Input
-    className='discovery-search'
-    prefix={<SearchOutlined />}
-    placeholder='Search studies by attribute...'
-    value={props.searchTerm}
-    onChange={props.handleSearchChange}
-    size='large'
-    allowClear
-  />
+const DiscoveryMDSSearch = (props: { searchTerm, handleSearchChange, inputSubtitle}) => (
+  <>
+    <Input
+      className='discovery-search'
+      prefix={<SearchOutlined />}
+      placeholder='Search studies by keyword...'
+      value={props.searchTerm}
+      onChange={props.handleSearchChange}
+      size='large'
+      allowClear
+    />
+    <div className='discovery-input-subtitle'>{props.inputSubtitle}</div>
```

Comment:
why don't put this as part of the `placeholder` msg for `Input`?

Reply:
That could be an alternative way to do this, for sure. But the ticket asked to put it below the box, I confirmed with PM Noah.

Reply:
(and, I'll wait until feat/download PR is merged before I merge this. I'll be pushing more UI tickets into this PR)